server/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -61,6 +63,40 @@ public void dropViews() {
         }
     }
 
+    @Test
+    public void debug_test_to_be_removed() {
+        execute("CREATE TABLE test1 (testid  INTEGER,  category INTEGER)");
+        execute("INSERT INTO  test1 (testid, category) VALUES (1,1), (2,2)");
+
+        execute("CREATE TABLE testcategories (category INTEGER,description TEXT)");
+        execute("INSERT INTO testcategories VALUES (1,'a'),(2,'b')");
+
+        refresh();
+
+        execute("CREATE VIEW vw_test AS\n" +
+            "SELECT test1.testid\n" +
+            "\t,testcategories.category AS aliasedcategoryid\n" +
+            "\t,testcategories.description\n" +
+            "FROM test1\n" +
+            "INNER JOIN testcategories ON test1.category = testcategories.category;");
+//        execute("SELECT * FROM GENERATE_SERIES(1, 2) CROSS JOIN vw_test LIMIT 100");
+//        assertThat(response.rowCount()).isEqualTo(4L);
+        execute("explain SELECT * FROM GENERATE_SERIES(1, 2) CROSS JOIN vw_test LIMIT 100");
+        assertThat(response.rows()).isEqualTo("ds");
```

Comment:
just temporal test and assertion to check that it fails because of different plan but not because of NPE